clients/include/testing_dot_strided_batched_ex.hpp

```diff
@@ -180,8 +180,31 @@ hipblasStatus_t testing_dot_strided_batched_ex_template(const Arguments& argus)
 
         if(argus.unit_check)
         {
-            unit_check_general<Tr>(1, batch_count, 1, h_cpu_result, h_hipblas_result_host);
-            unit_check_general<Tr>(1, batch_count, 1, h_cpu_result, h_hipblas_result_device);
+            if(std::is_same<Tr, hipblasHalf>{})
+            {
+                double tol = pow(2, -14) * N;
```

Comment:
Not sure but a helper function may be better, which can have comment on usage validity.   Could be put in near.xxx